pynucastro/rates/rate.py

```diff
@@ -391,6 +396,26 @@ def __init__(self, rfile=None, rfile_path=None, chapter=None, original_source=No
         if self.tabular:
             self.get_tabular_rate()
 
+    def modify_products(self, new_products):
+        if not isinstance(new_products, (set, list, tuple)):
+            new_products = [new_products]
+
+        self.products = []
+        for p in new_products:
+            if isinstance(p, Nucleus):
+                self.products.append(p)
+            else:
+                self.products.append(Nucleus(p))
+
+        self.modified = True
+
+        # we need to update the Q value and the print string for the rate
+
+        self._set_q()
+        self._set_screening()
```

Comment:
with self_screening you want to get rid of the neutrons?

Reply:
correct, if we are screening the reverse.  I just redid all the setup that looks at products

Reply:
ok, great!